Controls/Video.cs

```diff
@@ -7,6 +7,8 @@ namespace MissionPlanner.Controls
 {
     public partial class Video : Form
     {
+        private readonly GStreamer stream = new GStreamer();

```

Comment:
need to confirm if this can be instance based. i have a feeling it cant. as it will lead to native crashs

Reply:
I can open multiple GStreamer sources (multiple gimbal windows and a separate stream on the HUD) with no apparent issues if that's what you mean.